docs/en/advanced-features/kafka-reporter.md

```diff
@@ -0,0 +1,147 @@
+# Kafka Reporter
```

Comment:
This file needs to update, pointing to this new doc. 

https://github.com/apache/skywalking-go/blob/main/docs/menu.yml